web/pages/cart/cart.jsx

```diff
@@ -89,11 +104,26 @@ const Summary = ({ cart }) => {
 
 const Cart = () => {
   const { cart, isLoading } = useContext(CartContext)
-
+  const handleEmptyCart = async () => {
+    try {
+      cartService.emptyCart()
```

Comment:
this should be exposed from `CartContext`. That way there won't be a need to reload

Reply:
We'll do it in another PR. Im merging this for now